log4j-core/src/test/java/org/apache/logging/log4j/core/lookup/MapLookupTest.java

```diff
@@ -107,7 +107,7 @@ public void testLookupDefaultMapIsCheckedBeforeMapMessage() {
                 .setMessage(message)
                 .build();
         final MapLookup lookup = new MapLookup(map);
-        assertEquals("ADefault", lookup.lookup(event, "A"));
+        assertEquals("AEvent", lookup.lookup(event, "A"));
```

Comment:
Hi @quaff 
Thank you for your PR.
I think you should test both a key that maps to a default and another that does not.
You want to make sure the test covers all the paths in the lookup method. 
TY!

Reply:
@garydgregory Fixed.